### ISSUE TITLE
Jupyter Extension popup “Extension activation failed..."

### ISSUE THREAD

**Issue by ouliuquan** [eclipse-theia/theia-blueprint#247]
### Bug Description:
When I install the [Jupyter](https://open-vsx.org/extension/ms-toolsai/jupyter), and open the .py file, TheiaBlueprint always show the error message, I had excluded others extensions, confirmation that all the version of jupyter will make the UI popup error.

### Steps to Reproduce:

1.search `Jupyter` in Extension or download in [open-vsx](https://open-vsx.org/extension/ms-toolsai/jupyter)
2.open a Python file (*.py)
3.
![image](https://user-images.githubusercontent.com/38776890/207852659-50ba3db2-dc89-48c1-b47c-d017d6a775be.png)

### Additional Information

- Operating System:Windows 11, Python3.10.9
- Theia Version:1.32.0

**Comment by vince-fugnitto** [comment]
@ouliuquan thank you for reporting the issue.

I believe if you follow the notification and open the developer tools you'll see an error regarding `notebook` support as it is currently not implemented in the framework (https://github.com/eclipse-theia/theia/issues/8395#issuecomment-1348841020).

**Comment by vince-fugnitto** [comment]
In addition, there is no version of `jupyter` listed in the extensions-view given there is no compatible version, are you connected to the public instance https://open-vsx.org/, bumped the default api version or installed the extensions manually?

**Comment by ouliuquan** [comment]
@vince-fugnitto First of all thank you very much for your help. Excuse me I'm so used to using API 1.75.0 that I didn't realize it. I did upgrade the API, and then I tried the official 1.32.0 (API 1.55.2), and this extension is indeed not recommended.

I am currently doing research on enabling theia to support [Python-2021.4.765268190](https://github.com/microsoft/vscode-python/releases/tag/2021.4.765268190) (the last version without Pylance). I make a discussions on https://github.com/eclipse-theia/theia/discussions/11974, I have found that it can support up to this version without the Pylance, but I don't know why it rely on Jupyter, so I also make the open-vsx.org support full Jupyter's package https://github.com/open-vsx/publish-extensions/pull/656.

For now, there are still 2 question:
 * I hope that the prompt error can be resolved(this issue).
 * I hope that I can fix the `Run` button on 2021.4.765268190 version, the button is post on the discussions https://github.com/eclipse-theia/theia/discussions/11974
 
Thanks for your guidance, I learned a lot
